### PR TITLE
Update SSH Remoting demo text

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
Fixed header markdown
Moved links to bottom (they were being rendered incorrectly)
Adjusted formatting of `Setup Windows Machine` instructions to make it more readable
Fixed numbering of steps

### CODE REVIEW

File: demos/SSHRemoting/README.md [hunk 1]
```diff
@@ -14,67 +12,62 @@ The New-PSSession, Enter-PSSession and Invoke-Command cmdlets now have a new par
...
     - You can tell if it has the SSH remoting support by looking at the parameter sets for New-PSSession
-    ```powershell
-    Get-Command New-PSSession -syntax
-    New-PSSession [-Name <string[]>] [-HostName <string>] [-UserName <string>] [-KeyPath <string>] [<CommonParameters>]
-    ```
-1.  Install the latest [Win32 Open SSH] build from GitHub using the [installation] instructions
-1.  Edit the sshd_config file at the location where you installed Win32 Open SSH
+```powershell
+Get-Command New-PSSession -syntax
+New-PSSession [-Name <string[]>] [-HostName <string>] [-UserName <string>] [-KeyPath <string>] [<CommonParameters>]
```
Comment (PaulHigin) [inline demos/SSHRemoting/README.md]: The syntax is out of date after changes from cmdlet RFC.  
```
New-PSSession [-HostName] <string[]> [-Name <string[]>] [-UserName <string>] [-KeyFilePath <string>] [-SSHTransport] [<CommonParameters>]
```

**Review by PaulHigin — CHANGES_REQUESTED** [review]
LGTM except that the cmdlet syntax has changed slightly (see my inline comment on it).

File: demos/SSHRemoting/README.md (hunk 1 above)
Comment (SteveL-MSFT) [inline demos/SSHRemoting/README.md]: Changed it to syntax from alpha.17

**Review by PaulHigin — APPROVED** [review]
LGTM

**Pull request merged**